modules/partytown/config/PARTYTOWN_CONFIG.ts

```diff
@@ -0,0 +1,15 @@
+interface Partytown {
+  debug: boolean
+  forward: string[]
+  add: any
+}
+
+export default (): Partytown => {
```

Comment:
Why is partytown config a function? It can be plain object. Also move it do config/partytown/config.ts file